Lib/textwrap.py

```diff
@@ -114,6 +117,7 @@ class TextWrapper:
 
     def __init__(self,
                  width=70,
+                 cjk=False,
```

Comment:
These arguments can be passed as positional argument.
So new argument should be added last.